step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesBucketingHandler.java

```diff
@@ -66,6 +67,22 @@ public void processGauges(List<Measurement> measurements) {
         });
     }
 
+    /**
+     * This method will handle existing measurements, and will check if it is a gauge or normal measurement
+     * @param measurement
+     */
+    public void ingestExistingMeasurement(Measurement measurement) {
+        if (measurement == null) {
+            return;
+        }
+        measurement.remove("_id"); // because these measurements come with a generated id and can't be grouped into buckets.
```

Comment:
Removing the id could create side effects for the caller method. Would be better to ignore the Id durint the ingestion